cdap-app-fabric/src/main/java/co/cask/cdap/app/preview/PreviewModule.java

```diff
@@ -81,5 +83,8 @@ protected void configure() {
     bind(PreviewManager.class).to(DefaultPreviewManager.class).in(Scopes.SINGLETON);
 
     bind(Scheduler.class).to(NoopScheduler.class);
+
+    bind(PreviewStore.class).to(DefaultPreviewStore.class).in(Scopes.SINGLETON);
+    expose(PreviewStore.class);
```

Comment:
It is needed since we will have one injector for each preview run. So need to it to be exposed. The `PreviewManager` in line 83 also need to be exposed for same reason. Think the change will come later with the implementation of `DefaultPreviewManager`